nofos/nofos/templates/nofos/nofo_edit.html

```diff
@@ -58,144 +58,254 @@ <h1 class="font-heading- margin-0" title="{{ nofo_name_str }}">{{ nofo_name_str
+              id="{{ status_form.status.name }}"
+              onchange="this.form.submit()"
+            >
+              {% for value, label in status_form.status.field.choices %}
+                {% if value == '' %}
+                  <option disabled>{{ label }}</option>
+                {% else %}
+                  <option value="{{ value }}" {% if status_form.status.value == value %}selected{% endif %}>
+                    {{ label }}
+                  </option>
+                {% endif %}
+              {% endfor %}
+            </select>
+          </div>
+        </form>
+      {% else %}
+          NOFO is ‘Archived’.
       {% endif %}
-    </div>
-  </div>
-</div>
-{% elif nofo.status == 'published' and nofo.modifications %}
-<div class="usa-summary-box" role="region" aria-labelledby="summary-box-key-information">
-  <div class="usa-summary-box__body">
-    <h2 class="usa-summary-box__heading font-heading-sm" id="summary-box-key-information">
-      NOFO is ‘<a href="{% url 'nofos:nofo_edit_status' nofo.id %}">{{ nofo.get_status_display }}</a>’ with modifications.
-    </h2>
-    <div class="usa-summary-box__text">
-      <p>Make sure to update the “<a href="#modifications">Modifications</a>” section at the end of this NOFO with the changes you’ve made since publication.</p>
-      <p><a href="{% url 'nofos:nofo_history_modifications' nofo.id %}">See changes made since this NOFO was modified</a>.</p>
-      <p>Last modification date on cover page: <a href="{% url 'nofos:nofo_modifications' nofo.id %}">{{ nofo.modifications|date:"F j, Y" }}</a>.</p>
-    </div>
-  </div>
-</div>
-{% elif nofo.status == 'published' %}
-<div class="usa-summary-box" role="region" aria-labelledby="summary-box-key-information">
-  <div class="usa-summary-box__body">
-    <h2 class="usa-summary-box__heading font-heading-sm" id="summary-box-key-information">
-      NOFO is ‘<a href="{% url 'nofos:nofo_edit_status' nofo.id %}">{{ nofo.get_status_display }}</a>’.
-    </h2>
+    </h4>
+
     <div class="usa-summary-box__text">
-      <p>Ordinarily, published NOFOs can’t be edited, re-imported, or deleted. To edit this NOFO, you can <a href="{% url 'nofos:nofo_edit_status' nofo.id %}">change its
-        status</a>.</p>
+
+      <!-- Status-specific messages and actions -->
+      {% if nofo.archived %}
+        <!-- ARCHIVED NOFO -->
+        <p>Archived NOFOs exist as a record for posterity but can't be edited in any way. If you would like to restore this NOFO, please get in touch with Paul or Adam.</p>
+        {% if nofo.successor %}
+          <p>This is a past version of <a href="{% url 'nofos:nofo_edit' nofo.successor.id %}">{{ nofo.successor }}</a></p>
+        {% endif %}
+
+      {% elif nofo.status == 'cancelled' %}
+        <!-- CANCELLED NOFO -->
+        <p>Cancelled NOFOs exist as a record for posterity but can't be edited in any way. If you would like to restore this NOFO, change its status to something else using the dropdown above.</p>
```

Comment:
Let's remove "using the dropdown above" whenever we are using it.

We want this sentence to be as short as possible without losing meaning.